Library/Homebrew/cmd/uninstall.rb

```diff
@@ -50,6 +50,8 @@ def uninstall
       all_kegs:           args.force?,
     )
 
+    return if all_kegs.blank? && casks.blank?
```

Comment:
I'm a little confused by the need for this. At first glance, it looks like this should be made unnecessary by the `min: 1` arg to `named_args`.

Is it because the named args given might not resolve into any kegs/casks?

Reply:
> Is it because the named args given might not resolve into any kegs/casks?

Yep, exactly. If the args refer to kegs/casks that don't exist or are not installed and the `--force` flag is set, those resolution errors will be ignored because of the `ignore_unavailable` option. I assumed it made sense not to run `brew autoremove` if we knew for a fact that nothing had been uninstalled. I'll add a comment explaining the reasoning behind it.